doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -420,6 +423,151 @@ an additional feature and still be conformant with Anuket.
+Kubernetes currently only allows for one network, the *cluster* network and one network attachment for each pod. All pods and containers have an *eth0* interface, this interface is created by Kubernetes at pod creation and attached to the cluster network. All communication to and from the pod is done through this interface. To only allow for one interface in a pod removes the need for traditional networking tools such as *VRFs* and additional routes and routing tables inside the pod network namespace.
+
+The basic semantics of Kubernetes and the information found in manifest defines the connectivity rules and behavior without in principle any references to IP addresses. This has many advantages, it makes it easy to create portable, scalable SW services and network policies for them that are not location aware and therefore can be executed more or less anywhere.
+
+Kubernetes built in objects
+Pod and workloads | Description
+------------------|------------
+[Pod:](https://kubernetes.io/docs/concepts/workloads/pods/) | Pod is a collection of containers that can run on a host. This resource is created by clients and scheduled onto hosts.
+[ReplicaSet:](https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/) | ReplicaSet ensures that a specified number of pod replicas are running at any given time.
+[Deployment:](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) | Deployment enables declarative updates for Pods and ReplicaSets.
+[DaemonSet:](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/) | A Daemon set ensures that the correct nodes run a copy of a Pod.
+[Job:](https://kubernetes.io/docs/concepts/workloads/controllers/job/) | A Job represent a task, it creates one or more Pods and will continue to retry until the expected number of successful completions is reached.
+[CronJob:](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/) | A CronJob manages time-based Job, namely: once at a specified point in time repeatedly at a specified point in time
+[StatefulSet:](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) | StatefulSet represents a set of pods with consistent identities. Identities are defined as: network, storage. 
+
+Network objects | Description
+----------------|------------
+[Ingress:](https://kubernetes.io/docs/concepts/services-networking/ingress/) | Ingress is a collection of rules that allow inbound connections to reach the endpoints defined by a backend. An Ingress can be configured to give services externally reachable URLs, load balance traffic, terminate SSL, offer name based virtual hosting etc.
+[Service:](https://kubernetes.io/docs/concepts/services-networking/service/) | Service is a named abstraction of software service (for example, MySQL) consisting of local port (for example 3306) that the proxy listens on, and the selector that determines which pods will answer requests sent through the proxy.
```

Comment:
it doesn't have to be a "software service" -- it can be any application
```suggestion
[Service:](https://kubernetes.io/docs/concepts/services-networking/service/) | Service is a named abstraction of an application running on a set of pods consisting of a local port (for example 3306) that the proxy listens on, and the selector that determines which pods will answer requests sent through the proxy.
```